packages/clerk-js/src/core/resources/__tests__/__snapshots__/Environment.test.ts.snap

```diff
@@ -1,47 +1,49 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Environment has the same initial properties 1`] = `
-Environment {
-  "authConfig": AuthConfig {
-    "claimedAt": null,
-    "pathRoot": "",
-    "singleSessionMode": true,
+exports[`Environment __internal_toSnapshot() 1`] = `
```

Comment:
Note to reviewers: It seems like the new snapshot has inverted the function orders, thus the many changes